packages/discord.js/src/structures/PartialInteractionMessage.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { InteractionType } = require('discord-api-types/v9');
+
+class PartialInteractionMessage {
```

Comment:
Totally happy with what this is representing, but `MessageInteraction` isn't even a class, just a typedef for the `message.interaction` property.

This has the naming backwards, `InteractionMessage`, is defined as a "partial" class yet has a method that it's "full" equivalent can't have because it's just an object